tests/net/net_pkt/prj.conf

```diff
@@ -4,11 +4,13 @@ CONFIG_ZTEST_NEW_API=y
 CONFIG_NETWORKING=y
 CONFIG_NET_L2_ETHERNET=y
 CONFIG_NET_IPV4=y
-CONFIG_NET_IPV6=n
+CONFIG_NET_IPV6=y
 CONFIG_NET_ARP=y
 CONFIG_NET_UDP=y
 CONFIG_NET_BUF_LOG=y
 CONFIG_NET_LOG=y
 CONFIG_NET_BUF_POOL_USAGE=y
 CONFIG_ENTROPY_GENERATOR=y
 CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_NET_IPV6_NBR_CACHE=y
+CONFIG_NET_ETHERNET_BRIDGE=y
```

Comment:
Why do we need bridging enabled?

Reply:
I enabled it in order to test l2_bridged flag:
https://github.com/zephyrproject-rtos/zephyr/blob/d42d14e392e70a846abe5baaceb849ef21a50584/include/zephyr/net/net_pkt.h#L394

Reply:
Ok, was just wondering about that because it was not very clear.